### DISCUSSION TITLE
Create Cable_Types

### DISCUSSION THREAD

**Discussion by nbrgit** [netbox-community/netbox#10989] [Q&A]
Hey All,
can someone Explain how to creat my Own Cable Types?
I will use the Bulk Connector and this Application requieres Cable Types.

**Comment by mtinberg** [comment]
I believe these choices are statically defined in the software, but if there is another cable type that is missing it can be added by making a feature request detailing the entry you want.

I don't believe the cable type field is required though on a cable, not populating it is an option if it doesn't fit with what you are doing, or making a custom field on the cable model to add whatever data you find useful.

What is the cable type you think is missing?


—

Mark Tinberg ***@***.***>

Division of Information Technology-Network Services

University of Wisconsin-Madison


________________________________
From: nbrgit ***@***.***>
Sent: Monday, November 21, 2022 5:49 AM
To: netbox-community/netbox ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [netbox-community/netbox] Create Cable_Types (Discussion #10989)


Hey All,
can someone Explain how to creat my Own Cable Types?
I will use the Bulk Connector and this Application requieres Cable Types.

**Comment by nbrgit** [comment]
Thanks for the answer.

I try to make Bulk Connection with this small Software:
https://github.com/tiffanykaczmarek/netbox-bulk-connect

In an earlier Version of Netbox I’ve add in a file Cabletypes and it works, but this was just a test installation… now I want to integrate Netbox in Our System.
With a ne Installation and Doesn’t know where I’ve made the Changes…

Maybe you know this tool or know with the Code what is to do 😊

Thanks a lot!

Grüße aus der Grünen Hölle / Regards from the Green Hell

i. A. Dennis Schüller
Systembetreuung
IT

***@***.***

T +49 (2691) 302 9885
M +49 151 571 320 36
F +49 2691 302 9897

Nürburgring 1927
GmbH & Co. KG

Otto-Flimm-Straße
53520 Nürburg
nuerburgring.de

***@***.***


***@***.***<https://nuerburgring.de/shop/231?category_id=231&page=1>

Bitte schonen Sie unsere Umwelt und drucken die Email nur aus, wenn es wirklich notwendig ist!
Please consider the environment before printing this email!


Von: Mark Tinberg ***@***.***>
Gesendet: Montag, 21. November 2022 15:56
An: netbox-community/netbox ***@***.***>
Cc: Schüller Dennis ***@***.***>; Author ***@***.***>
Betreff: Re: [netbox-community/netbox] Create Cable_Types (Discussion #10989)

I believe these choices are statically defined in the software, but if there is another cable type that is missing it can be added by making a feature request detailing the entry you want.

I don't believe the cable type field is required though on a cable, not populating it is an option if it doesn't fit with what you are doing, or making a custom field on the cable model to add whatever data you find useful.

What is the cable type you think is missing?


—

Mark Tinberg ***@***.***<mailto:***@***.***>>

Division of Information Technology-Network Services

University of Wisconsin-Madison


________________________________
From: nbrgit ***@***.***<mailto:***@***.***>>
Sent: Monday, November 21, 2022 5:49 AM
To: netbox-community/netbox ***@***.***<mailto:***@***.***>>
Cc: Subscribed ***@***.***<mailto:***@***.***>>
Subject: [netbox-community/netbox] Create Cable_Types (Discussion #10989)


Hey All,
can someone Explain how to creat my Own Cable Types?
I will use the Bulk Connector and this Application requieres Cable Types.

—
Reply to this email directly, view it on GitHub<https://github.com/netbox-community/netbox/discussions/10989>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AAS7UM42O5BVUAQJXNWIGIDWJNOT7ANCNFSM6AAAAAASGRZX2Q>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***<mailto:***@***.***>>


—
Reply to this email directly, view it on GitHub<https://github.com/netbox-community/netbox/discussions/10989#discussioncomment-4196392>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AHZRGZ6G3RB6RURTBG5T6W3WJOEOJANCNFSM6AAAAAASGRZX2Q>.
You are receiving this because you authored the thread.Message ID: ***@***.******@***.***>>



--

Diese Mail wurde auf Computerviren geprüft

**Reply by candlerb** [reply]
> I try to make Bulk Connection with this small Software: https://github.com/tiffanykaczmarek/netbox-bulk-connect

I don't know that tool, but now that Netbox has custom scripts you can use this script instead:
https://github.com/netbox-community/reports/blob/master/scripts/multi_connect.py

As for cable types: these are not user-changeable, and as @mtinberg said, if you find you need a cable type other than the ones listed, you should create a feature request on github for this.

I'm not going to tell you where this list is in the source code; if you change the source code, then you have forked Netbox and you are responsible for maintaining your fork going forwards.  I _strongly_ recommend you don't do this.  There are plenty of ways to customize Netbox without modifying the code (e.g. custom fields, scripts, reports, plugins ...)

**Comment by nbrgit** [comment]
Thank you for the Script! 😊
I’ll test it 😊

Grüße aus der Grünen Hölle / Regards from the Green Hell

i. A. Dennis Schüller
Systembetreuung
IT

***@***.***

T +49 (2691) 302 9885
M +49 151 571 320 36
F +49 2691 302 9897

Nürburgring 1927
GmbH & Co. KG

Otto-Flimm-Straße
53520 Nürburg
nuerburgring.de

***@***.***


***@***.***<https://nuerburgring.de/shop/231?category_id=231&page=1>

Bitte schonen Sie unsere Umwelt und drucken die Email nur aus, wenn es wirklich notwendig ist!
Please consider the environment before printing this email!


Von: Brian Candler ***@***.***>
Gesendet: Mittwoch, 23. November 2022 19:04
An: netbox-community/netbox ***@***.***>
Cc: Schüller Dennis ***@***.***>; Author ***@***.***>
Betreff: Re: [netbox-community/netbox] Create Cable_Types (Discussion #10989)


I try to make Bulk Connection with this small Software: https://github.com/tiffanykaczmarek/netbox-bulk-connect

I don't know that tool, but now that Netbox has custom scripts you can use this script instead:
https://github.com/netbox-community/reports/blob/master/scripts/multi_connect.py

As for cable types: these are not user-changeable, and as @mtinberg<https://github.com/mtinberg> said, if you find you need a cable type other than the ones listed, you should create a feature request on github for this.

I'm not going to tell you where this list is in the source code; f you change the source code, then you have forked Netbox and you are responsible for maintaining your fork going forwards. I strongly recommend you don't do this. There are plenty of ways to customize Netbox without modifying the code (e.g. custom fields, scripts, reports, plugins ...)

—
Reply to this email directly, view it on GitHub<https://github.com/netbox-community/netbox/discussions/10989#discussioncomment-4219048>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AHZRGZ2ZAE4AVCM4IAHWRRDWJZMBXANCNFSM6AAAAAASGRZX2Q>.
You are receiving this because you authored the thread.Message ID: ***@***.******@***.***>>



--

Diese Mail wurde auf Computerviren geprüft

**Comment by nbrgit** [comment]
Hey All,
today, I’ve Upgrade my Netbox to v3.4.0
Now I get this error:
***@***.***

Could someone help me? 😊

Thanks a Lot!


Grüße aus der Grünen Hölle / Regards from the Green Hell

i. A. Dennis Schüller
Systembetreuung
IT

***@***.***

T +49 (2691) 302 9885
M +49 151 571 320 36
F +49 2691 302 9897

Nürburgring 1927
GmbH & Co. KG

Otto-Flimm-Straße
53520 Nürburg
nuerburgring.de

***@***.***


***@***.***<https://nuerburgring.de/shop/231?category_id=231&page=1>

Bitte schonen Sie unsere Umwelt und drucken die Email nur aus, wenn es wirklich notwendig ist!
Please consider the environment before printing this email!


Von: Schüller Dennis
Gesendet: Donnerstag, 24. November 2022 07:53
An: 'netbox-community/netbox' ***@***.***>
Betreff: AW: [netbox-community/netbox] Create Cable_Types (Discussion #10989)

Thank you for the Script! 😊
I’ll test it 😊

Grüße aus der Grünen Hölle / Regards from the Green Hell

i. A. Dennis Schüller
Systembetreuung
IT

***@***.******@***.***>

T +49 (2691) 302 9885
M +49 151 571 320 36
F +49 2691 302 9897

Nürburgring 1927
GmbH & Co. KG

Otto-Flimm-Straße
53520 Nürburg
nuerburgring.de

***@***.***


***@***.***<https://nuerburgring.de/shop/231?category_id=231&page=1>

Bitte schonen Sie unsere Umwelt und drucken die Email nur aus, wenn es wirklich notwendig ist!
Please consider the environment before printing this email!


Von: Brian Candler ***@***.******@***.***>>
Gesendet: Mittwoch, 23. November 2022 19:04
An: netbox-community/netbox ***@***.******@***.***>>
Cc: Schüller Dennis ***@***.******@***.***>>; Author ***@***.******@***.***>>
Betreff: Re: [netbox-community/netbox] Create Cable_Types (Discussion #10989)


I try to make Bulk Connection with this small Software: https://github.com/tiffanykaczmarek/netbox-bulk-connect

I don't know that tool, but now that Netbox has custom scripts you can use this script instead:
https://github.com/netbox-community/reports/blob/master/scripts/multi_connect.py

As for cable types: these are not user-changeable, and as @mtinberg<https://github.com/mtinberg> said, if you find you need a cable type other than the ones listed, you should create a feature request on github for this.

I'm not going to tell you where this list is in the source code; f you change the source code, then you have forked Netbox and you are responsible for maintaining your fork going forwards. I strongly recommend you don't do this. There are plenty of ways to customize Netbox without modifying the code (e.g. custom fields, scripts, reports, plugins ...)

—
Reply to this email directly, view it on GitHub<https://github.com/netbox-community/netbox/discussions/10989#discussioncomment-4219048>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AHZRGZ2ZAE4AVCM4IAHWRRDWJZMBXANCNFSM6AAAAAASGRZX2Q>.
You are receiving this because you authored the thread.Message ID: ***@***.******@***.***>>



--

Diese Mail wurde auf Computerviren geprüft

**Reply by candlerb** [reply]
> today, I’ve Upgrade my Netbox to v3.4.0 Now I get this error: `***@***.***` Could someone help me? 😊

Erm, I don't think the error you're getting is literally `***@***.***`.  Can you show the actual error?

Having said this, it doesn't surprise me.  Netbox's APIs and data models have no stability guarantees between versions: clients of REST API, plugins, custom scripts and reports may all need to change to match the new APIs.  Therefore if you use any of these things, it's very important that you try them out in a test environment *before* you upgrade, and if they don't work, modify them to work with the new version.
